app/controllers/api/v1/api_docs_controller.rb

```diff
@@ -113,6 +113,7 @@ class ApiDocsController < ActionController::Base
           Api::V1::SearchController,
           Api::V1::Visualization::ClustersController,
           Api::V1::Visualization::AnnotationsController,
+          Api::V1::Visualization::ExpressionController,
```

Comment:
previously, we had excluded this from api docs since these endpoints were expected to be volatile and change a lot as, e.g. we make performance improvements, and so we didn't want people scripting against it.  
If we're adding it back, we should add a comment in the docs about that volatility

Reply:
I mainly added this for debugging purposes while I was doing this work as it made it easier for me to iterate.  I'm happy to add the comment though, unless we really feel that these shouldn't be exposed in Swagger.

Reply:
1926bcf

Reply:
works for me